lib/chef/provisioning/chef_provider_action_handler.rb

```diff
@@ -42,16 +42,23 @@ def report_progress(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
     end
 
     def performed_action(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
```

Comment:
@tyler-ball `resource_updated` is only supposed to be called *after* all the converge_bys complete.  Like, typically, it goes:

```ruby
<start action>
resource_update_applied("set content of '/x/y.txt' to 'hello world")
resource_update_applied("set mode of '/x/y.txt' to 0666)
resource_updated(my_file_resource)
```